gossip/src/crds_gossip_pull.rs

```diff
@@ -1441,7 +1441,7 @@ pub(crate) mod tests {
         caller: &CrdsValue,
         num_items: usize,
     ) {
-        let packet_data_size_range = (PACKET_DATA_SIZE - 5)..=PACKET_DATA_SIZE;
+        let packet_data_size_range = (PACKET_DATA_SIZE - 7)..=PACKET_DATA_SIZE;
```

Comment:
How is this change related to the PR?

Reply:
This is testing that the pull requests "almost" fill up a packet.
There will be some slack because the serialized size of the pull-request will be a step function.
The test is asserting that that slack is within reasonable range (so that we don't erroneously leave half of the packet wasted).
As described in https://github.com/anza-xyz/agave/pull/4505/files#r1920126073 
ContactInfo::new_localhost has changed a little bit, so this needs to be updated accordingly.